iml-services/iml-device/src/main.rs

```diff
@@ -165,9 +165,11 @@ async fn update_client_mounts(
                 mountpoint = Null,
                 state = 'unmounted',
                 state_modified_at = now()
-            WHERE filesystem != ALL ($1)
+            WHERE host_id = $1
+            AND id != ALL($2)
```

Comment:
As I understand, `filesystem` vs `id` shouldn't matter, the former is more descriptive, but the latter is shorter :)

Reply:
That would only be the case if (filesystem, host_id) was guaranteed to be unique, but it's not.